packages/email/email.js

```diff
@@ -227,6 +231,7 @@ Email.customTransport = undefined;
  * @param {Object[]} [options.attachments] Array of attachment objects, as
  * described in the [nodemailer documentation](https://nodemailer.com/message/attachments/).
  * @param {MailComposer} [options.mailComposer] A [MailComposer](https://nodemailer.com/extras/mailcomposer/#e-mail-message-fields)
+ * @deprecated in 2.8
```

Comment:
I don't think the deprecation notice here makes sense, since eventually all the async functions will be merged into the regular ones.

Reply:
I thought we want to go with flow like this:
1. Only sync.
2. Both sync and async (with a suffix).
3. Only async (with a suffix).
4. Only async (without a suffix).

If so, then I'd mark them as deprecated. If not, then the flow looks like this:
1. Only sync.
2. Both sync and async (with a suffix).
3. Only async (without a suffix).

And we don't need a deprecation.

Reply:
The correct strategy is:

1. Only sync. (Current)
2. Both sync and async (with a suffix).
3. Only async (without a suffix).

Thank you! @StorytellerCZ and @radekmie